### ISSUE TITLE
Think about case studies

### ISSUE THREAD

**Issue by keller-mark** [hms-dbmi/cistrome-higlass-wrapper#174]
I think it would be a useful exercise to find a paper or two which use one of the existing tools (UCSC, Wash U epigenome browser) to test hypotheses related to gene regulation (ideally using some Cistrome DB data), and try to reproduce the result in our tool, and add this as a demo.

**Comment by sehilyi** [comment]
It would be very helpful if we can find such papers.

**Comment by keller-mark** [comment]
I agree. There are many results on google scholar for papers that cite the Cistrome and CistromeDB papers
 - Mei et al 2016 https://scholar.google.com/scholar?cites=4410185230290837678&as_sdt=40000005&sciodt=0,22&hl=en 
- Zheng et al 2019 https://scholar.google.com/scholar?cites=14138633253049970899&as_sdt=40000005&sciodt=0,22&hl=en
- Liu et al 2011 https://scholar.google.com/scholar?cites=17374025607959613119&as_sdt=40000005&sciodt=0,22&hl=en

but a better approach may be to just ask our collaborators about the best ones they know

**Comment by keller-mark** [comment]
In addition we may want to prioritize reproducing the Mei et al 2016 "Data visualization and extensions" section, figure 2E

paper section:
> Cistrome DB also provides visualization functions that allow users to view peaks and signal intensity in either the UCSC (25) or WashU (26) genome browsers. Visualization of both single or batch samples is supported. For example, a ‘super-enhancer’ region of the genome contains multiple SOX2 and NANOG binding sites and is enriched in mediator and H3K27ac signals (Figure 2E). Using Cistrome DB, users can select the relevant ChIP-seq samples and visualize the co-binding pattern between master transcription factors on the WashU genome browser using the sample batch view function (Figure 2E)

figure 2E caption
> Batch sample visualization through WashU browser showing the co-binding pattern between master transcription factors in embryonic stem cells


<img width="568" alt="Screen Shot 2020-03-09 at 11 25 09 AM" src="https://user-images.githubusercontent.com/7525285/76229525-bbdfa000-61f8-11ea-8049-e3eed9512c35.png">

**Comment by keller-mark** [comment]
Perhaps this is also a good use case for the "add track that only contains a selected row" feature

**Comment by keller-mark** [comment]
One suggestion from Cliff: look at developmental genes that are regulated differently in different cell types (for example, MYC)
